doc-gen/package.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "ak-docgen",
+    "version": "3.18.2",
+    "description": "A CLI tool used to generate .mdx documentation for jsdoc public methods",
+    "author": "Alchemy",
+    "license": "MIT",
+    "private": true,
+    "type": "module",
+    "main": "./dist/esm/exports/index.js",
+    "types": "./dist/types/exports/index.d.ts",
+    "typings": "./dist/types/exports/index.d.ts",
+    "sideEffects": false,
+    "files": [
+      "dist",
+      "src/**/*.ts",
+      "!dist/**/*.tsbuildinfo",
+      "!vitest.config.ts",
+      "!.env",
+      "!src/**/*.test.ts",
+      "!src/__tests__/**/*"
+    ],
+    "exports": {
+      ".": {
+        "types": "./dist/types/exports/index.d.ts",
+        "default": "./dist/esm/exports/index.js"
+      },
+      "./package.json": "./package.json"
+    },
+    "bin": {
+      "ak-docgen": "./dist/esm/cli.js"
+    },
+    "scripts": {
+      "prebuild": "node --loader ts-node/esm ./inject-version.ts",
+      "build": "yarn clean && yarn build:esm && yarn build:types",
+      "build:esm": "tsc --project tsconfig.build.json --module es2015 --outDir ./dist/esm --removeComments && echo > ./dist/esm/package.json '{\"type\":\"module\"}'",
+      "build:types": "tsc --project tsconfig.build.json --module esnext --declarationDir ./dist/types --emitDeclarationOnly --declaration --declarationMap",
+      "postbuild": "chmod +x ./dist/esm/cli.js",
+      "clean": "rm -rf ./dist"
+    },
+    "devDependencies": {
+      "@types/fs-extra": "^11.0.4",
+      "typescript": "^5.0.4",
+      "typescript-template": "*",
+      "vitest": "^0.31.0"
+    },
+    "publishConfig": {
+      "access": "public",
+      "registry": "https://registry.npmjs.org/"
+    },
+    "repository": {
+      "type": "git",
+      "url": "git+https://github.com/alchemyplatform/aa-sdk.git"
+    },
+    "bugs": {
+      "url": "https://github.com/alchemyplatform/aa-sdk/issues"
+    },
+    "homepage": "https://github.com/alchemyplatform/aa-sdk#readme",
+    "dependencies": {
+      "bundle-require": "^4.0.2",
+      "cac": "^6.7.14",
+      "change-case": "^5.4.3",
+      "chokidar": "^3.6.0",
+      "dedent": "^1.5.1",
+      "esbuild": "^0.20.1",
+      "find-up": "^7.0.0",
+      "fs-extra": "^11.2.0",
+      "ora": "^8.0.1",
+      "pathe": "^1.1.2",
+      "picocolors": "^1.0.0",
+      "typescript": "^5.0.4"
+    }
+  }
```

Comment:
(Nit) The indentation on this curly brace is upsetting.

Reply:
XD I couldn't see it for so long! will def fix it